LICENSE

```diff
@@ -0,0 +1,437 @@
+Attribution-NonCommercial-ShareAlike 4.0 International
```

Comment:
The change to the plaintext version of the license is just to satisfy vsce, since it looks for a plaintext license file.